duckduckgo/duckduckgo-autofill
# Show grayscale Dax icon for in-context signup

**Reviewer:** @shakyShane @GioSensation 
**Asana:** https://app.asana.com/0/0/1204038361216228/f

## Description
Show a greyscale icon for in-context signup

## Steps to test
1. Load branch autofill into extension (branch [`abrown/bugfix-default-pixel-params`](https://github.com/duckduckgo/duckduckgo-privacy-extension/compare/abrown/bugfix-default-pixel-params)) and enable feature toggle
2. Go to https://duckduckgo.com/newsletter
3. Confirm in-context signup shows with gray Dax icon
4. Log in to Email Protection and confirm Dax is now in color

In-context signup | Email Protection
--- | ---
<img width="2160" alt="Screenshot 2023-02-27 at 18 43 52" src="https://user-images.githubusercontent.com/635903/221655048-d9be53a3-01a7-4987-a38b-8a815dc1242f.png"> | <img width="2160" alt="Screenshot 2023-02-27 at 18 44 11" src="https://user-images.githubusercontent.com/635903/221654996-2456dd7f-4a3c-4188-a964-51dfede07a39.png">
